Python/_warnings.c

```diff
@@ -431,6 +431,45 @@ call_show_warning(PyObject *category, PyObject *text, PyObject *message,
     return -1;
 }
 
+static int
+tb_displayline(PyObject *f, PyObject *filename, int lineno, PyObject *name)
+{
+    int err;
+    PyObject *line;
+
+    if (filename == NULL || name == NULL)
+        return -1;
+    line = PyUnicode_FromFormat("  File \"%U\", line %d, in %U\n",
+                                filename, lineno, name);
+    if (line == NULL)
+        return -1;
+    err = PyFile_WriteObject(line, f, Py_PRINT_RAW);
+    Py_DECREF(line);
+    if (err != 0)
+        return err;
+    /* ignore errors since we can't report them, can we? */
```

Comment:
I suggest to remove the comment.

We could report the error, but I don't think that it's worth it. It would be annoying that logging a warning would emit a second warning/exception.